routes/index.js

```diff
@@ -35,24 +37,20 @@ router.post('/token', async function(req, res) {
   res.json({token: token});
 });
 
-function getDefaultJwtClaim() {
-  var index = process.argv.indexOf("--claims");
-  var args = process.argv.slice(index + 1);
-  if (args.length > 0) {
-    return JSON.parse(args[0]);
-  } else {
-    return {"username": "test@test.com", "userId": 1, "authorities": ["AUTH_1"]};
-  }
-}
+function getCommandLineArg(flag, defaultValue) {
```

Comment:
could you help to change this function to return the raw value passed from the command line. so don't do JSON.parse here. each argument has its own type. we can do the parse when the argument is used.